code/test/sanityCheck.py

```diff
@@ -24,28 +35,29 @@ def checkRxnAnnotation(rxns):
     """
     check consistency of rxn lists between model and annotation file
     """
-    rxnAssoc = pd.read_table("model/reactions.tsv")
-    rxnList = rxnAssoc['rxns'].to_list()
+    rxnList = get_column_from_tsv("model/reactions.tsv", "rxns")
+    spontaneous = get_column_from_tsv("model/reactions.tsv", "spontaneous", False)
+    rxnDeprecated = get_column_from_tsv("data/deprecatedIdentifiers/deprecatedReactions.tsv", "rxns")
+    assert not bool(set(rxnList) & set(rxnDeprecated)), "Deprecated reaction reused!"
     assert rxnList == rxns, "Reaction annotation mismatch!"
-    assert pd.api.types.is_numeric_dtype(
-        rxnAssoc['spontaneous']), "Spontaneous column should be in numeric!"
+    assert pd.api.types.is_numeric_dtype(spontaneous), "Spontaneous column should be in numeric!"
 
 
 def checkMetAnnotation(mets):
     """
     check consistency of met lists between model and annotation file
     """
-    metAssoc = pd.read_table("model/metabolites.tsv")
-    metList = metAssoc['mets'].to_list()
+    metList = get_column_from_tsv("model/metabolites.tsv", "mets")
+    metDeprecated = get_column_from_tsv("data/deprecatedIdentifiers/deprecatedMetabolites.tsv", "mets")
+    assert not bool(set(metList) & set(metDeprecated)), "Deprecated metabolite reused!"
```

Comment:
Same thinking as above:

```suggestion
    assert set(metList).isdisjoint(set(metDeprecated)), "Deprecated metabolite reused!"
```